### DISCUSSION TITLE
Abfallplus not working

### DISCUSSION THREAD

**Discussion by AndiHaas85** [mampfes/hacs_waste_collection_schedule#2038] [General]
I had the Integration with de.k4systems.abfallapploe working, but i don't get any waste schedules since last week. 
So i tried the examples, e.g. 

waste_collection_schedule:
    sources:
    - name: app_abfallplus_de
      args:
        app_id: de.albagroup.app
        city: Braunschweig
        strasse: Hauptstraße
        hnr: 7A

But even with this i don't get any dates.

When i try with 

  sources:
    - name: abfallnavi_de
      args:
        service: coe
        ort: Coesfeld
        strasse: Ahornweg

all waste dates are available. So I don't think it's a Problem with my configuration. 

Sensor is defined in sensors.yaml an this is included in configuration.yaml

- platform: waste_collection_schedule
  name: muell

**Comment by 5ila5** [comment]
Duplicate of #1989
